src/main/kotlin/biokotlin/featureTree/Enums.kt

```diff
@@ -0,0 +1,87 @@
+package biokotlin.featureTree
+
+/**
+ * Represents the strand of a gff feature.
+ * @property gffName the string that defines a given strand within a gff file
+ */
+enum class Strand(val gffName: String) {
+    /**
+     * Positive strand of the DNA.
+     */
+    PLUS("+"),
+
+    /**
+     * Negative strand of the DNA.
+     */
+    MINUS("-"),
+
+    /**
+     * Features that do not have a strand (eg scaffolds).
+     */
+    NOT_STRANDED("."),
```

Comment:
Why both NOT_STRANDED and UNKNOWN.  When do you expect each to be used?

Reply:
Per [Gff3 file spec](https://github.com/The-Sequence-Ontology/Specifications/blob/master/gff3.md):

> Column 7: "strand"
    The strand of the feature. + for positive strand (relative to the landmark), - for minus strand, and . for features that are not stranded. In addition, ? can be used for features whose strandedness is relevant, but unknown.

I am not exactly sure of the instances where someone would want to disambiguate between "." and "?", but I want to ensure that if they do (which is permissible in the spec) that they are able to generate the same output as the input. I believe "." is to be used for things that lack a strand because the underlying biology is not stranded while "?" is for things that contain a strand, but that strand is unknown.

Reply:
ok  - that makes sense